public/dashboard.php

```diff
@@ -7,12 +7,21 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Dashboard</title>
+    <title>dashboard</title>
+    <link rel="stylesheet" href="dashboard.css">
 </head>
 <body>
+    <div class="navbar">
+        <div class="navlinks">
+            <ul class="active">
+                <li><a href="">profile</a></li>
+                <li><a href="">skills</a></li>
+                <li><a href="">certificatins</a></li>
+                <li><a href="">project</a></li>
+                <li><a href="">logout</a></li>
 
-
-    <h1>Welcome</h1>
-    
+            </ul>
+        </div>
+    </div>
```

Comment:
is it only the navbar that is suppose to be done on the dasbhoard, 
you are suppose to include other peoples works

eg

`include profile.php`
`include skills.php`
.
.